### PR TITLE
Inactive notifications in BAM Cloud

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

Inactive notifications in BAM Cloud

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [x] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: cloud/service-mapping/ba-management.md [hunk 1]
```diff
@@ -215,8 +214,7 @@ In Centreon Cloud, when you create a business activity, you must always add it t
 
 #### Notification
 
-Configure when and for what status notifications should be sent when the BA's
-status changes. Add the **Notification** contact group so that users can be notified.
+This feature is not currently active.
```
Comment (nmagniezCentreon) [inline cloud/service-mapping/ba-management.md:217]: mettre un commentaire du genre 
"BAs use common notification mechanism see [here ](https://docs.centreon.com/cloud/alerts-notifications/notif-configuration/)for more details

File: cloud/service-mapping/ba-settings.md [hunk 2]
```diff
@@ -36,20 +36,14 @@ The default parameters below define values for BAs.
...
+| Warning Threshold        | *Warning* threshold: Corresponds to a warning condition of the BA.  |
+| Critical Threshold       | *Critical* threshold: Corresponds to a critical condition of the BA. |
 | Reporting Period         | Time Default BA reporting time period                                                                 |
-| Notification Time Period | Default BA notification time period                                                                   |
-| Contact Groups           | Default BA notified contact groups                                                                    |
-| Notification Interval    | Default BA notification interval                                                                      |
+| Notification Time Period | This option is currently inactive                                                                   |
+| Contact Groups           | This option is currently inactive                                                                   |
+| Notification Interval    | This option is currently inactive                                                                   |
```
Comment (nmagniezCentreon) [inline cloud/service-mapping/ba-settings.md:44]: same as above

**Review by nmagniezCentreon — CHANGES_REQUESTED** [review]

File: cloud/service-mapping/ba-management.md (hunk 1 above)
Comment (cg-tw) [inline cloud/service-mapping/ba-management.md:217]:
```suggestion
This screen is not currently active. In Centreon Cloud, BAs use [the same notification mechanism as hosts and services](https://docs.centreon.com/cloud/alerts-notifications/notif-configuration/).
```

File: cloud/service-mapping/ba-settings.md [hunk 3]
```diff
@@ -36,20 +36,14 @@ The default parameters below define values for BAs.
...
-| Warning Threshold        | *Warning* threshold: Corresponds to a warning condition of the BA. A notification can then be sent.   |
-| Critical Threshold       | *Critical* threshold: Corresponds to a critical condition of the BA. A notification can then be sent. |
+| Warning Threshold        | *Warning* threshold: Corresponds to a warning condition of the BA.  |
+| Critical Threshold       | *Critical* threshold: Corresponds to a critical condition of the BA. |
 | Reporting Period         | Time Default BA reporting time period                                                                 |
-| Notification Time Period | Default BA notification time period                                                                   |
-| Contact Groups           | Default BA notified contact groups                                                                    |
-| Notification Interval    | Default BA notification interval                                                                      |
+| Notification Time Period | This option is currently inactive                                                                   |
```
Comment (cg-tw) [inline cloud/service-mapping/ba-settings.md:42]:
```suggestion
| Notification Time Period | This option is currently inactive. In Centreon Cloud, BAs use [the same notification mechanism as hosts and services](https://docs.centreon.com/cloud/alerts-notifications/notif-configuration/).                                                                  |
```

File: cloud/service-mapping/ba-settings.md [hunk 4]
```diff
@@ -36,20 +36,14 @@ The default parameters below define values for BAs.
...
+| Contact Groups           | This option is currently inactive                                                                   |
```
Comment (cg-tw) [inline cloud/service-mapping/ba-settings.md:43]:
```suggestion
| Contact Groups           | This option is currently inactive. In Centreon Cloud, BAs use [the same notification mechanism as hosts and services](https://docs.centreon.com/cloud/alerts-notifications/notif-configuration/).                                                                  |
```

File: cloud/service-mapping/ba-settings.md (hunk 2 above)
Comment (cg-tw) [inline cloud/service-mapping/ba-settings.md:44]:
```suggestion
| Notification Interval    | This option is currently inactive. In Centreon Cloud, BAs use [the same notification mechanism as hosts and services](https://docs.centreon.com/cloud/alerts-notifications/notif-configuration/).                                                                   |
```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/service-mapping/ba-management.md [hunk 5]
```diff
@@ -210,8 +207,7 @@ Dans Centreon Cloud, lorsque vous crééz une activité métier, vous devez syst
 
 #### Notification
 
-Configurez quand et pour quels statuts des notifications doivent être envoyées lorsque le statut de l'activité métier change.
-Ajoutez le groupe de contacts **Notification** pour que les utilisateurs puissent être notifiés.
+Cette fonctionnalité n'est pas active actuellement.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/service-mapping/ba-management.md:210]:
```suggestion
Cet écran n'est pas actif actuellement. Dans Centreon Cloud, les BA utilisent [le même mécanisme de notification que les hôtes et les services](https://docs.centreon.com/cloud/alerts-notifications/notif-configuration/).
```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/service-mapping/ba-settings.md [hunk 6]
```diff
@@ -40,26 +40,18 @@ pré-remplir les champs lors de la bascule de la configuration en mode
...
-| Groupes de contact      | Groupes de contact notifiés par défaut de la BA                                                        |
-| Intervalle entre deux   | Intervalle de notification par défaut de la BA                                                         |
-
-Pour être en mesure d'envoyer des notifications aux utilisateurs ou
-solutions externes (Slack, OpenDuty etc..) appartenant au(x) groupe(s)
-de contacts sélectionnés, vous devez configurer la section "Business
-activity" pour les utilisateurs concernés :
-
-![image](../assets/service-mapping/guide/bam_user_notification.png)
+| Période de notification | Option actuellement inactive                                                          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/service-mapping/ba-settings.md:52]:
```suggestion
| Période de notification | Option actuellement inactive. Dans Centreon Cloud, les BA utilisent [le même mécanisme de notification que les hôtes et les services](https://docs.centreon.com/cloud/alerts-notifications/notif-configuration/).                                                          |
```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/service-mapping/ba-settings.md [hunk 7]
```diff
@@ -40,26 +40,18 @@ pré-remplir les champs lors de la bascule de la configuration en mode
...
+| Groupes de contact      | Option actuellement inactive                                                        |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/service-mapping/ba-settings.md:53]:
```suggestion
| Groupes de contact      | Option actuellement inactive. Dans Centreon Cloud, les BA utilisent [le même mécanisme de notification que les hôtes et les services](https://docs.centreon.com/cloud/alerts-notifications/notif-configuration/).                                                       |
```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/service-mapping/ba-settings.md [hunk 8]
```diff
@@ -40,26 +40,18 @@ pré-remplir les champs lors de la bascule de la configuration en mode
...
+| Intervalle entre deux   | Option actuellement inactive                                                           |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/service-mapping/ba-settings.md:54]:
```suggestion
| Intervalle entre deux   | Option actuellement inactive. Dans Centreon Cloud, les BA utilisent [le même mécanisme de notification que les hôtes et les services](https://docs.centreon.com/cloud/alerts-notifications/notif-configuration/).                                                          |
```

**Review by nmagniezCentreon — APPROVED** [review]

**Pull request merged**